sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/TelemetryItem.cs

```diff
@@ -25,18 +25,21 @@ internal partial class TelemetryItem
 
             if (activity.GetTelemetryType() == TelemetryType.Request)
             {
-                if (activityTagsProcessor.activityType.HasFlag(OperationType.V2))
+                var displayName = activity.DisplayName;
+                if (!string.IsNullOrEmpty(displayName)
+                    && !displayName.StartsWith(activity.Source.Name))
+                {   // AspNetCore instrumentation sets DisplayName to the same value we want for OperationName
+                    // Also, it's important than any user-set DisplayName be used #44971
+                    Tags[ContextTagKeys.AiOperationName.ToString()] = displayName.Truncate(SchemaConstants.Tags_AiOperationName_MaxLength);
+                }
+                else if (activityTagsProcessor.activityType.HasFlag(OperationType.V2))
                 {
                     Tags[ContextTagKeys.AiOperationName.ToString()] = TraceHelper.GetOperationNameV2(activity, ref activityTagsProcessor.MappedTags).Truncate(SchemaConstants.Tags_AiOperationName_MaxLength);
```

Comment:
Note that the logic in `TraceHelper.GetOperationNameV2` (and `TraceHelper.GetOperationNameV2`) sets the same value that `Activity.DisplayName` was already set to by the AspNetCore instrumentation. So unless this exporter is used _without_ OpenTelemetry.Instrumentation.AspNetCore or `.AddAspNetCoreInstrumentation()` (which currently isn't possible) lines 28-37 (in the old code) are redundant with the instrumentation library. 

For this PR I left them in, because there are tests that enforce this logic, but I think they should be removed.